<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>streamMaskDialog</class>
 <widget class="QDialog" name="streamMaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Files By Mask</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>571</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lbStreamMask">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream File &amp;Mask:</string>
    </property>
    <property name="buddy">
     <cstring>lbStreamMask</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBrowse">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="valueRangeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Use Mask Value Range:</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QSpinBox" name="eIndex">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Start Index:</string>
     </property>
     <property name="buddy">
      <cstring>sIndex</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;End Index:</string>
     </property>
     <property name="buddy">
      <cstring>eIndex</cstring>
     </property>
    </widget>
    <widget class="QSpinBox" name="sIndex">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lbStreamMask</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>streamMaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>streamMaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
